ops/dev-deploy.tmpl.yaml

```diff
@@ -114,28 +126,10 @@ extraEnvVars: &envVars
     value: hyku-dev-redis-master
   - name: REDIS_URL
     value: redis://:$REDIS_PASSWORD@hyku-dev-redis-master:6379/adventist
-  - name: SETTINGS__ACTIVE_JOB__QUEUE_ADAPTER
-    value: good_job
-  - name: SETTINGS__BULKRAX__ENABLED
-    value: "true"
-  - name: SETTINGS__CONTACT_EMAIL
-    value: support@notch8.com
-  - name: SETTINGS__DEVISE__INVITATION_FROM_EMAIL
-    value: support@notch8.com
-  - name: SETTINGS__FITS_PATH
-    value: /app/fits/fits.sh
-  - name: SETTINGS__MULTITENANCY__ADMIN_ONLY_TENANT_CREATION
-    value: "true"
-  - name: SETTINGS__MULTITENANCY__DEFAULT_HOST
-    value: "%{tenant}.s2.adventistdigitallibrary.org"
-  - name: SETTINGS__MULTITENANCY__ENABLED
```

Comment:
I'm curious if there's there's an equivalent HYKU setting for this? Or does it just go away now?

Reply:
they go away